src/lib/email.ts

```diff
@@ -62,12 +62,14 @@ const sendEmailWithSES = async ({
     },
   };
   const ses = new SES({
-    region: "us-east-2",
-    credentials: {
-      accessKeyId: env.AWS_ACCESS_KEY_ID,
-      secretAccessKey: env.AWS_SECRET_ACCESS_KEY,
-    },
+    region: env.AWS_REGION,
+    credentials:
```

Comment:
could you elaborate a bit more here as well? Thanks

Reply:
@itsjoeoui Credentials are automatically retrieved from EC2 instance when they are not provided explicitly.